src/iwant_bot/start.py

```diff
@@ -79,10 +79,7 @@ def check_post_request(body):
 
     if trigger_word == 'repeat':
         res = web.json_response(body=format_response(user_name + ' wants me to ' + text))
-        print(res.headers)
-        print(res.body)  # how to display full response body? It is <aiohttp.payload.StringPayload object at 0x7fc9e266c2e8>
         return res
-        #return web.json_response(body=format_response(user_name + ' wants me to ' + text))
     elif trigger_word == 'whoami':
         return web.json_response(body=format_response('You are ' + user_name + ' with id ' + user_id))
```

Comment:
Actually this line doesn't have to be so long - I would propose:
- Usage of formatted strings s.a. `f'You are {user_name} with id {user_id}'`.
- Having two statements (on two lines) instead of a single one - `response = f'You are ...'`, `return web.json_response(body=format_response(response))`